VTEX - Giftcard API.json

```diff
@@ -1253,153 +1251,163 @@
-                    "relationName": {
-                        "type": "string",
-                        "nullable": true,
-                        "default": "insert name here"
-                    },
-                    "redemptionCode": {
-                        "type": "string",
-                        "nullable": true,
-                        "default": "example code"
-                    },
-                    "discounts": {
-                        "type": "integer",
-                        "format": "int32",
-                        "default": 0
-                    },
-                    "shipping": {
-                        "type": "integer",
-                        "format": "int32",
-                        "default": 0
-                    },
-                    "taxes": {
-                        "type": "integer",
-                        "format": "int32",
-                        "default": 12
-                    },
-                    "items": {
-                        "type": "array",
-                        "items": {
-                            "$ref": "#/components/schemas/Item"
-                        },
-                        "description": ""
-                    },
-                    "itemsTotal": {
-                        "type": "integer",
-                        "format": "int32",
-                        "default": 100
-                    }
-                },
-                "example": {
-                    "grandTotal": 123.1,
-                    "relationName": null,
-                    "redemptionCode": null,
-                    "discounts": 0,
-                    "shipping": 0,
-                    "taxes": 12,
-                    "items": [
-                        {
-                            "productId": "1",
-                            "id": "1",
-                            "refId": "12",
-                            "name": "Product Name",
-                            "price": 100,
-                            "quantity": 1
-                        }
-                    ],
-                    "itemsTotal": 100
-                }
-            },
+				"title": "Cart information.",
+				"required": [
+					"grandTotal",
+					"relationName",
+					"redemptionCode",
+					"discounts",
+					"shipping",
+					"taxes",
+					"items",
+					"itemsTotal"
+				],
+				"type": "object",
+				"properties": {
+					"grandTotal": {
+						"type": "integer",
+                        "description": "Total payment value.",
+                        "example": 182
+					},
+					"relationName": {
+						"type": "string",
+						"nullable": true,
+                        "description": "Represents the relationship between the client and the store.",
+                        "example": null
+					},
+					"redemptionCode": {
+						"type": "string",
+						"default": "AH52F7",
+						"description": "Minimum of six characters",
+                        "example": "XTYB"
```

Comment:
- Doesn't it have a minimum of six characters?

Reply:
Adjusted.